src/modules/esl-traversing-query/README.md

```diff
@@ -6,17 +6,18 @@ Authors: *Alexey Stsefanovich (ala'n)*.
 
 <a name="intro"></a>
 
-**ESLTraversingQuery** is a utility to find elements via an extended selector query.
+**ESLTraversingQuery** is a utility module that extends the capabilities of CSS selectors for traversing and selecting elements in the Document Object Model (DOM). It provides an enhanced querying syntax that allows for more flexible and powerful element selection, traversal, and filtering.
 
 Extended query supports:
  - plain CSS selectors
- - relative selectors  
+ - relative selectors
 NOTE: Selectors that don't start from a plain selector will use passed base Element as a root
  - `::next` and `::prev` sibling pseudo-selectors
  - `::parent`, `::closest` and `::child` pseudo-selectors
  - `::find` pseudo-selector
  - `::first`, `::last` and `:nth(#)` limitation pseudo-selectors
  - `::visible` visible element pseudo-selector
+ - `::not`, `::filter` filtration pseudo-selectors
```

Comment:
as a suggestion - do we need to specify here which ones we just apply and which ones we pass "arguments" to

Reply:
I don't think so, from examples, it's quite obvious. We can discuss it  on the Monday call